docs/migrating_sse.md

```diff
@@ -41,7 +41,7 @@ Here is a table of a mapping from SSE intrinsics to their LSX counterpart (WIP):
 | _mm_blend_epi16         |                                       |
 | _mm_blend_pd            |                                       |
 | _mm_blend_ps            |                                       |
-| _mm_blendv_epi8         |                                       |
+| _mm_blendv_epi8         | __lsx_vbitsel_v                       |
```

Comment:
There is a slight difference: __blendv_epi8 only checks the MSB of each byte, however __lsx_vbitsel_v is bitwise.

Reply:
thank you this is my mistake